ss_admin_console/main/default/classes/SSACFilter.cls

```diff
@@ -1,162 +1,144 @@
 public with sharing class SSACFilter {
-	private String sObjectApiName;
-    private Map<String, SObject> records;
-    private List<String> keyFields;
-    private List<String> allFields;
-    private String additionCondition;
-	private List<SObject> constantSObjects;
+    private SObjectType sObjectApiName;
+    private List<SObject> records;
+    private List<SObjectField> keyFields;
+    private List<SObjectField> allFields;
+    private List<SObject> constantSObjects;
     private Map<String, Set<Object>> constantFields;
```

Comment:
Do you need this field? It's used only to build a SOQL. So you can store the values as s method local variable